libnavui-androidauto/build.gradle

```diff
@@ -40,8 +40,9 @@ dependencies {
 
     // This defines the minimum version of Navigation which is included in this SDK. To upgrade the
     // Navigation versions, you can specify a newer version in your downstream build.gradle.
-    releaseApi("com.mapbox.navigation:android:2.9.0-alpha.4")
-    debugApi project(":libnavigation-android")
```

Comment:
Am removing this, it does not catch important runtime errors. Every runtime error this exposes is an issue we need to address.

Reply:
We may add a `androidauto-app` to keep it separated. But I'm still a hesitant on that because we are discovering actionable problems with the exposure.

Reply:
Actually @Zayankovsky keeps bringing up a good point that this issue will happen often with Search

so it'll block us more than it's worth